src/components/MusicPage/MusicPage.js

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components/macro'
+import {
+  ShowFormatWarmup,
+  ShowFormatIntervalsT,
+  ShowFormatCooldown,
+} from '../WorkoutCard/ShowFormatFromSections'
+import { selectColor, selectColorByInterval } from '../../services/selectColor'
+import Button from '../Button'
+import Icon from 'supercons'
+import { NavLink } from 'react-router-dom'
+
+export default function MusicPage({ location }) {
+  const { name, warmup, intervalsT, cooldown } = location.workout
+
+  return (
+    <Grid>
+      <Title>
+        {name} <br />
+      </Title>
+
+      <Wrapper>
+        <div>
+          <WarmupSection background={selectColor(warmup.powerHigh)}>
+            <ShowFormatWarmup warmup={warmup} />
+          </WarmupSection>
+          <IntervalsTSection
+            background={selectColorByInterval(intervalsT.onPower)}
+          >
+            <ShowFormatIntervalsT intervalsT={intervalsT} />
+          </IntervalsTSection>
+          <CooldownSection background={selectColor(cooldown.powerHigh)}>
+            <ShowFormatCooldown cooldown={cooldown} />
+          </CooldownSection>
+        </div>
+      </Wrapper>
+
+      <BackButton as={NavLink} to={{ pathname: '/' }}>
```

Comment:
I think you can write it like this, according to the react router dom documentation.
```suggestion
      <BackButton as={NavLink} path="/" >
```

Reply:
I am currently using the ToLocation option from React Router and if I just write path= , it shows me this error: 


resolveToLocation(to, currentLocation),
  52 |   currentLocation
  53 | );
> 54 | const { pathname: path } = toLocation;
     | ^  55 | // Regex taken from: https://github.com/pillarjs/path-to-regexp/blob/master/index.js#L202
  56 | const escapedPath =
  57 |   path && path.replace(/([.+*?=^!:${}()[\]|/\\])/g, "\\$1");

I left it as it was.